tests/Fixer/PhpTag/NoClosingTagFixerTest.php

```diff
@@ -121,9 +121,12 @@ function bar()
                 '<?php ?>',
             ),
             'No code, only comment' => array(
-                '<?php /* license */ ',
+                '<?php /* license */',
```

Comment:
very well...
then commit log will be directly passed into project branch log. please adjust msg-log to be meaningful